tests/schemas/pilot.schema.json

```diff
@@ -10,6 +10,13 @@
     "xws": { "type": "string", "pattern": "^[a-z0-9-]+$" },
     "ability": { "type": "string" },
     "text": { "type": "string" },
+    "standardLoadout": { "type": "boolean"},
+    "upgrades": {
```

Comment:
Please put `upgrades` and `slots` inside a `OneOf` so that only one of them can be present. There's an example of `OneOf` in `shipstats` I think

Reply:
We also discussed adding some conditional logic around `upgrades` and `standardLoadout`, but I think there might be a cleaner solution. What if we remove the `standardLoadout` boolean and rename the `upgrades` array to `standardLoadout`? That's effectively what we were going for: only ships that have `standardLoadout` have `upgrades`.

So the resulting schema would look something like:
```
...
OneOf{
  "slots": blahblahblah list of slots,
  "standardLoadout": blahblahblah list of upgrades
}
...
```

Reply:
I like combining them, it really is duplicate information. Good idea!

Reply:
Because these are both top level properties and not nested inside another, the oneof has to be in the same place as required to my knowledge. Will tag you in the update to make sure it looks correct.